test/OpenTelemetry.Exporter.Geneva.Tests/GenevaMetricExporterTests.cs

```diff
@@ -301,12 +303,12 @@ public void DisableMetricNameValidationTest(bool disableMetricNameValidation)
                 {
                     if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
                     {
-                        options.ConnectionString = $"Account=OTelMonitoringAccount;Namespace=OTelMetricNamespace;DisableMetricNameValidation={disableMetricNameValidation}";
+                        options.ConnectionString = $"Account=OTelMonitoringAccount;Namespace=OTelMetricNamespace;DisableMetricNameValidation={disableMetricNameValidation};PrivatePreviewOtlpProtobufMetricExporter={enableOtlpProtobufexporter}";
```

Comment:
Could you conditionally append the string with the opt-in key instead of always having it in the connection string? This test was originally written to test ConnectionStrings that don't have this new opt-in key.

```csharp
var connectionString = $"Account=OTelMonitoringAccount;Namespace=OTelMetricNamespace;DisableMetricNameValidation={disableMetricNameValidation}";

if (enableOtlpProtobufexporter)
{
 connectionString = connectionString + "PrivatePreviewOtlpProtobufMetricExporter=true";
}
```

Reply:
sure, I can do that.